sprokit/src/sprokit/pipeline/process_factory.h

```diff
@@ -193,6 +194,49 @@ kwiver::vital::plugin_factory_vector_t const& get_process_list();
                                                  typeid( sprokit::process ).name(), \
                                                  sprokit::create_new_process< proc_type > ) )
 
+// ============================================================================
+class process_registrar
+  : public kwiver::plugin_registrar
+{
+public:
+  enum option {
```

Comment:
Possibly overengineering at this point (i.e. something we can add later, at need), but this smells like a candidate for a flags type.